requirements_dev.txt

```diff
@@ -1,4 +1,5 @@
 -r requirements.txt
 
+numpy>=1.10.0
```

Comment:
It's just in the dev requirements so the tests run and it's going to be a long time before numpy hits another major... This is the standard line we use everywhere else; I don't think it's beneficial to change it here, nor do I think this is the place to have such a wide reaching conversation about what limits to put on numpy. If numpy manages to break the api for the fundamental `np.array` function, I'm cool with this failing.